### PR TITLE
Ignite 2.9 release notes

### PR DESCRIPTION
**Pull request by alex-plekhanov**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: RELEASE_NOTES.txt [hunk 1]
```diff
@@ -2,6 +2,194 @@ Apache Ignite Release Notes
 ===========================
 
 
+Apache Ignite In-Memory Computing Platform 2.9
+-----------------------------------------------------------
+
+Ignite Core:
+* Added feature of cluster snapshots for persistence caches
```
Comment (dmagda) [inline RELEASE_NOTES.txt:9]: Alex, I can't say for the docs completeness but would encourage us to run the text through Grammarly or a similar tool that can catch misprints.

For instance, we need to add the "a" or "the" article before "feature", "ability", etc. Also, Grammarly can help to rephrase some sentences.

Comment (alex-plekhanov) [reply RELEASE_NOTES.txt]: Thanks, fixed. Could you please have a look again?

File: RELEASE_NOTES.txt [hunk 2]
```diff
@@ -2,6 +2,194 @@ Apache Ignite Release Notes
...
+* Added the feature of cluster snapshots for persistence caches
+* Added tracing of transactions, discovery, exchange, and communication components
+* Added new cluster-wide state: read-only. In this state caches available for cache read operations. Cache operations with data modification (update, remove, clear, create, destroy and etc.) don't allowed
+* Added TDE master key rotation feature
+* Added functionality to run user-defined code inside a sandbox
+* Added Cluster ID and tag properties to identify the cluster
+* Added support for one-way thick-client to server connections
+* Added the ability to not open server sockets (for incoming connections) by thick client
+* Added management API to cancel user-provided tasks and queries
+* Added partition states system view
```
Comment (alamar) [inline RELEASE_NOTES.txt:18]: Maybe we could join the next 2 by two in a line?

File: RELEASE_NOTES.txt [hunk 3]
```diff
@@ -2,6 +2,194 @@ Apache Ignite Release Notes
...
+* Added OffHeapEntriesCount, OffHeapBackupEntriesCount, OffHeapPrimaryEntriesCount, HeapEntriesCount, CacheSize to cache metrics
+* Added query pool starvation monitoring
+* Added the ability to put non-primitive data types via HTTP-REST
+* Added the ability to use CACHE_CREATE and CACHE_DESTROY permissions on per-cache level
+* Added the ability to restrict access to the internal package of Ignite
+* Added nodes caching in cluster groups
+* Added warning of deadlock when an unordered collection is passed to putAll-style bulk cache operation
+* Added the ability to set user attributes for thin clients connections
+* Added a new public API for the cluster state change (activation/deactivation and etc.)
+* Added EVT_BASELINE_... events
```
Comment (alamar) [inline RELEASE_NOTES.txt:30]: Maybe we could group the next 3?

**Review by alamar — APPROVED** [review]
It's already LGTM but I have put some improvements.

File: RELEASE_NOTES.txt (hunk 2 above)
Comment (alex-plekhanov) [reply RELEASE_NOTES.txt]: Thanks, fixed

File: RELEASE_NOTES.txt (hunk 3 above)
Comment (alex-plekhanov) [reply RELEASE_NOTES.txt]: Thanks, fixed

**Pull request closed without merging**